test/test-client.js

```diff
@@ -338,6 +338,95 @@ describe('Imgix client:', function describeSuite() {
   });
 
   describe('Calling buildSrcSet()', function describeSuite() {
+    describe('with no parameters', function describeSuite() {
+      var srcset = new ImgixClient({
+        domain: 'testing.imgix.net',
+        includeLibraryParam: false,
+        secureURLToken: 'MYT0KEN'
+      }).buildSrcSet('image.jpg');
+      
+      it('should generate the expected default srcset pair values', function testSpec(){
+        resolutions = [100, 116, 134, 156, 182, 210, 244, 282,
+                       328, 380, 442, 512, 594, 688, 798, 926,
+                       1074, 1246, 1446, 1678, 1946, 2258, 2618,
+                       3038, 3524, 4088, 4742, 5500, 6380, 7400, 8192];
+        srclist = srcset.split(",");
+        src = srclist.map(function (srcline){
+          return parseInt(srcline.split(" ")[1].slice(0,-1), 10);
+        })
+
+        for (var i = 0; i < srclist.length; i++) {
+          assert.equal(src[i], resolutions[i]);
+        }
+      });
+
+      it('should return the expected number of `url widthDescriptor` pairs', function testSpec() {
+        assert.equal(srcset.split(',').length, 31);
+      });
+
+      it('should not exceed the bounds of [100, 8192]', function testSpec() {
+        var srcsetSplit = srcset.split(",");
+        var min = Number.parseFloat(
+          srcsetSplit[0]
+          .split(" ")[1]
+          .slice(0, -1)
+        );
+        var max = Number.parseFloat(
+          srcsetSplit[srcsetSplit.length-1]
+          .split(" ")[1]
+          .slice(0, -1)
+        );
+        assert(min >= 100);
+        assert(max <= 8192);
+      });
+
+      it('should not increase more than 18% every iteration', function testSpec() {
```

Comment:
This should be 16%, right?

Reply:
It should but looks like with the way we're currently rounding, a lot of values come out a little over 16%.